ubuntudesign/www.ubuntu.com
# updating legal short-form

## Done

* updated short name of Landscape On Premises to Landscape Product
* moved old page to archive 2016-05-20.html
* added old page to TOC on index
* added title to another archive that was missing it

## QA

1. go to /legal/short-terms and see that it matches the [copy doc](https://docs.google.com/document/d/1W0WJrXaIvC54MDPgS62xfgPhqJaSE7RcZ2mjIqz1Y2o/edit) only section 3.4 was updated

![image](https://cloud.githubusercontent.com/assets/441217/16577841/6136f522-428f-11e6-9717-2237facbb844.png)
2. see that /legal/short-terms/2016-05-20 is in the 'older versions section' and has a 'latest version on it'
3. see that the titles are all ok 

## Issue / Card

Fixes [bug 1596606](https://bugs.launchpad.net/canonical-website-content/+bug/1596606)
